controller/service.rst

```diff
@@ -104,8 +104,9 @@ defined above with the id ``app.hello_controller``::
 
 .. note::
 
-    You cannot drop the ``Action`` part of the method name when using this
-    syntax.
+    Unlike the traditional ``Bundle:Controller:method`` notation, when using
+    this syntax the method name must match the original controller method name,
+    so you cannot remove the ``Action`` suffix if the method includes it.
```

Comment:
Maybe this wording?

```rst
Make sure the method name in your route (e.g. ``indexAction``) matches the method name exactly.
Unlike the traditional ``Bundle:Controller:method`` notation, the `Action` suffix is not automatically
added for you.
```

Reply:
I like your proposal. Much better. Thanks!